types/consumer-data-standards/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for non-npm package consumer-data-standards-browser 1.0
+// Type definitions for non-npm package consumer-data-standards-browser 1.1
 // Project: https://github.com/ConsumerDataStandardsAustralia
 // Definitions by: CDR-API-Stream <https://github.com/ConsumerDataStandardsAustralia>
```

Comment:
Following the email exchange:

```suggestion
// Type definitions for non-npm package consumer-data-standards-browser 1.1
// Project: https://github.com/ConsumerDataStandardsAustralia
// Definitions by: Tomas Schier <https://github.com/tom-schier>
```

This will make future PRs much quicker.